aws_cloudtrail_eventbridge/cloudtrail-eventbridge.yaml

```diff
@@ -0,0 +1,177 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog AWS Cloudtrail Eventbridge Logs Integration
+Parameters:
+  ApiKey:
+    Description: >-
+      Your Datadog API Key
+    Type: String
+    AllowedPattern: .+
+    ConstraintDescription: ApiKey is required
```

Comment:
Can we make this noecho? Or does that not work for stacksets (haven't tried noecho + stacksets)?